### ISSUE TITLE
Single episode incorrectly identified as a Season Pack during manual search

### ISSUE THREAD

**Issue by lhbdhr** [Sonarr/Sonarr#8359]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Description
I am encountering a bug where a single episode is incorrectly processed as a "Season Pack" only *after* the download is triggered. 

During a manual search, Sonarr correctly identifies the release as a single episode (based on my custom scoring for season packs). 

However, once the download is sent to the client, Sonarr treats it as a full season, adding every episode of the series to the activity queue.

### Actual Behavior
- Immediately after clicking download, Sonarr adds tasks for **all** episodes in the series to the activity queue.
- After the download completes, the queue is filled with duplicate errors:  
  `Episode file already imported at 01/24/2026 19:42:46`.
- I must manually remove these redundant tasks from the activity queue to continue tracking new episodes.

### Additional Context
The issue seems to occur during the transition from "Search Result" to "Grabbed Task." Even though the manual search UI recognizes the correct format, the backend logic switches it to a season pack upon grabbing.

### Expected Behavior

Only the selected episode (S01E01) should be added to the activity queue and download client.

### Steps To Reproduce

1. Navigate to a series page (e.g., `https://xxxx.com/series/river-sunset`).
2. Click the **Manual Search** (person icon) for a specific episode (e.g., S01E01).
3. Select a release that is clearly a single episode, such as:  
   `River Sunset S01E01 1080p TX WEB-DL AAC2.0 H.264-MWeb`
   *Note: At this stage, Sonarr's scoring correctly reflects it as a single episode, not a season pack.*
4. Click the **Download** button.

### Environment

```markdown
- **Sonarr Version:** [4.0.16.2944]
- **Package Version:** [linuxserver.io]
- **Operating System:** [ Docker]
- **Indexer:** [Prowlarr]
```

### What branch are you running?

Main

### Trace Logs?

2026-01-24 20:34:58.2|Debug|ReleaseSearchService|Total of 2 reports were found for [River Sunset : S01E02] from 3 indexers
2026-01-24 20:34:58.2|Debug|ReleaseSearchService|Setting last search time to: 01/24/2026 12:34:58
2026-01-24 20:34:58.2|Info|DownloadDecisionMaker|Processing 2 releases
......
2026-01-24 20:34:58.3|Debug|DownloadDecisionMaker|Release 'River Sunset S01E02 2160p TX WEB-DL AAC2.0 H.265-MWeb' from 'M-Team - TP (Prowlarr)' rejected for the following reasons: [Permanent] 903.5 MB is smaller than minimum allowed 1.5 GB (for 45min), [Permanent] WEBDL-2160p is not wanted in profile
2026-01-24 20:34:58.3|Trace|DownloadDecisionMaker|Processing release 2/2
2026-01-24 20:34:58.3|Debug|DownloadDecisionMaker|Processing release 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb' from 'M-Team - TP (Prowlarr)'
2026-01-24 20:34:58.3|Debug|Parser|Parsing string 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb'
2026-01-24 20:34:58.3|Trace|Parser|^(?<title>.+?)(?:(?:[-_\W](?<![()\[!]))+S?(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))(?:[ex]|\W[ex]){1,2}(?<episode>\d{2,3}(?!\d+))(?:(?:\-|[ex]|\W[ex]|_){1,2}(?<episode>\d{2,3}(?!\d+)))*)(?:[-_. ]|$)
2026-01-24 20:34:58.3|Debug|Parser|Episode Parsed. River Sunset - S01E02 
2026-01-24 20:34:58.3|Debug|Parser|Languages parsed: Unknown
2026-01-24 20:34:58.3|Debug|QualityParser|Trying to parse quality for 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb'
2026-01-24 20:34:58.3|Debug|Parser|Quality parsed: WEBDL-1080p v1
2026-01-24 20:34:58.3|Debug|Parser|Release Group parsed: MWeb
2026-01-24 20:34:58.3|Debug|AggregateLanguages|Language couldn't be parsed from release, fallback to series original language: Chinese
2026-01-24 20:34:58.3|Debug|AggregateLanguages|Selected languages: Chinese
2026-01-24 20:34:58.3|Trace|DownloadDecisionMaker|Custom Format Score of '1700' [WEB Tier 01] calculated for 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb'
2026-01-24 20:34:58.3|Debug|AcceptableSizeSpecification|Beginning size check for: River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb
2026-01-24 20:34:58.3|Debug|AcceptableSizeSpecification|Item: River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb, meets size constraints
2026-01-24 20:34:58.3|Debug|AlreadyImportedSpecification|Performing already imported check on report
2026-01-24 20:34:58.3|Debug|AlreadyImportedSpecification|Skipping already imported check for episode without file
2026-01-24 20:34:58.3|Trace|CustomFormatAllowedbyProfileSpecification|Custom Format Score of 1700 [WEB Tier 01] above Series profile minimum 0
2026-01-24 20:34:58.3|Trace|ConfigService|Using default config value for 'skipfreespacecheckwhenimporting' defaultValue:'False'
2026-01-24 20:34:58.3|Trace|ConfigService|Using default config value for 'minimumfreespacewhenimporting' defaultValue:'100'
2026-01-24 20:34:58.3|Trace|ConfigService|Using default config value for 'maximumsize' defaultValue:'0'
2026-01-24 20:34:58.3|Debug|MaximumSizeSpecification|Maximum size is not set.
2026-01-24 20:34:58.3|Debug|MinimumAgeSpecification|Not checking minimum age requirement for non-usenet report
2026-01-24 20:34:58.3|Debug|QualityAllowedByProfileSpecification|Checking if report meets quality requirements. WEBDL-1080p v1
2026-01-24 20:34:58.3|Debug|ReleaseRestrictionsSpecification|Checking if release meets restrictions: River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb
2026-01-24 20:34:58.3|Debug|ReleaseRestrictionsSpecification|[River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb] No restrictions apply, allowing
2026-01-24 20:34:58.3|Debug|RetentionSpecification|Not checking retention requirement for non-usenet report
2026-01-24 20:34:58.3|Debug|SceneMappingSpecification|No applicable scene mapping, skipping.
2026-01-24 20:34:58.3|Debug|SeriesSpecification|Checking if series matches searched series
2026-01-24 20:34:58.3|Debug|DelaySpecification|Ignoring delay for user invoked search
2026-01-24 20:34:58.3|Debug|HistorySpecification|Skipping history check during search
2026-01-24 20:34:58.3|Debug|MonitoredEpisodeSpecification|Skipping monitored check during search
2026-01-24 20:34:58.3|Trace|ConfigService|Using default config value for 'autounmonitorpreviouslydownloadedepisodes' defaultValue:'False'
2026-01-24 20:34:58.3|Debug|DownloadDecisionMaker|Release 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb' from 'M-Team - TP (Prowlarr)' accepted
2026-01-24 20:34:58.4|Trace|Http|Res: 6341 [GET] /api/v3/release?episodeId=1639: 200.OK (2207 ms)
2026-01-24 20:34:58.4|Debug|Api|[GET] /api/v3/release?episodeId=1639: 200.OK (2207 ms)
2026-01-24 20:35:03.1|Trace|Http|Req: 6342 [POST] /api/v3/release (from 104.*.*.236 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36)
2026-01-24 20:35:03.1|Debug|Sonarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2026-01-24 20:35:03.2|Trace|HttpClient|Req: [GET] http://prowlarr:9696/1/download?apikey=(removed)&link=&file=River+Sunset+S01E02+1080p+TX+WEB-DL+AAC2.0+H.264-MWeb
2026-01-24 20:35:03.2|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:04.0|Trace|HttpClient|Res: HTTP/1.1 [GET] http://prowlarr:9696/1/download?apikey=(removed)&link=&file=River+Sunset+S01E02+1080p+TX+WEB-DL+AAC2.0+H.264-MWeb: 200.OK (8225 bytes) (778 ms)
2026-01-24 20:35:04.0|Debug|QBittorrent|Downloading torrent for episode 'River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb' finished (8225 bytes from http://prowlarr:9696/1/download?apikey=(removed)&link=&file=River+Sunset+S01E02+1080p+TX+WEB-DL+AAC2.0+H.264-MWeb)
2026-01-24 20:35:04.1|Trace|HttpClient|Req: [GET] http://qbittorrent:8080/api/v2/app/webapiVersion
2026-01-24 20:35:04.1|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:04.1|Trace|HttpClient|Res: HTTP/1.1 [GET] http://qbittorrent:8080/api/v2/app/webapiVersion: 200.OK (6 bytes) (5 ms)
2026-01-24 20:35:04.1|Trace|HttpClient|Response content (6 bytes): 2.11.2
2026-01-24 20:35:04.1|Trace|HttpClient|Req: [POST] http://qbittorrent:8080/api/v2/torrents/add: 
torrents=River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb.torrent (8225 bytes)
category=sonarr
stopped=False
ratioLimit=3
2026-01-24 20:35:04.1|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:04.2|Trace|HttpClient|Res: HTTP/1.1 [POST] http://qbittorrent:8080/api/v2/torrents/add: 200.OK (3 bytes) (136 ms)
2026-01-24 20:35:04.2|Trace|HttpClient|Response content (3 bytes): Ok.
2026-01-24 20:35:04.2|Info|DownloadService|Report sent to qBittorrent. Indexer M-Team - TP (Prowlarr). River Sunset S01E02 1080p TX WEB-DL AAC2.0 H.264-MWeb
2026-01-24 20:35:04.2|Trace|EventAggregator|Publishing EpisodeGrabbedEvent
2026-01-24 20:35:04.2|Trace|EventAggregator|EpisodeGrabbedEvent -> NotificationService
2026-01-24 20:35:04.2|Debug|NotificationService|Notification and series have one or more intersecting tags.
2026-01-24 20:35:04.2|Trace|EventAggregator|EpisodeGrabbedEvent <- NotificationService
2026-01-24 20:35:04.2|Trace|EventAggregator|EpisodeGrabbedEvent -> HistoryService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- HistoryService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> DownloadMonitoringService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- DownloadMonitoringService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> PendingReleaseService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- PendingReleaseService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> DownloadHistoryService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- DownloadHistoryService
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> CutoffController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- CutoffController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> MissingController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- MissingController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> EpisodeController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- EpisodeController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent -> CalendarController
2026-01-24 20:35:04.3|Trace|EventAggregator|EpisodeGrabbedEvent <- CalendarController
2026-01-24 20:35:04.3|Trace|Http|Res: 6342 [POST] /api/v3/release: 200.OK (1171 ms)
2026-01-24 20:35:04.3|Debug|Api|[POST] /api/v3/release: 200.OK (1171 ms)
2026-01-24 20:35:08.6|Trace|Http|Req: 6343 [GET] /api/v3/queue?page=1&pageSize=20&sortDirection=descending&sortKey=title&includeUnknownSeriesItems=true (from 104.*.*.236 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36)
2026-01-24 20:35:08.6|Debug|Sonarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2026-01-24 20:35:08.6|Trace|Http|Res: 6343 [GET] /api/v3/queue?page=1&pageSize=20&sortDirection=descending&sortKey=title&includeUnknownSeriesItems=true: 200.OK (31 ms)
2026-01-24 20:35:08.6|Debug|Api|[GET] /api/v3/queue?page=1&pageSize=20&sortDirection=descending&sortKey=title&includeUnknownSeriesItems=true: 200.OK (31 ms)
2026-01-24 20:35:09.3|Trace|CommandQueueManager|Publishing RefreshMonitoredDownloads
2026-01-24 20:35:09.3|Trace|CommandQueueManager|Checking if command is queued or started: RefreshMonitoredDownloads
2026-01-24 20:35:09.3|Trace|CommandQueueManager|Inserting new command: RefreshMonitoredDownloads
2026-01-24 20:35:09.3|Trace|CommandExecutor|RefreshMonitoredDownloadsCommand -> DownloadMonitoringService
2026-01-24 20:35:09.3|Trace|CommandQueueManager|Marking command as started: RefreshMonitoredDownloads
2026-01-24 20:35:09.3|Trace|HttpClient|Req: [GET] http://qbittorrent:8080/api/v2/app/webapiVersion
2026-01-24 20:35:09.3|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:09.3|Trace|HttpClient|Res: HTTP/1.1 [GET] http://qbittorrent:8080/api/v2/app/webapiVersion: 200.OK (6 bytes) (3 ms)
2026-01-24 20:35:09.3|Trace|HttpClient|Response content (6 bytes): 2.11.2
2026-01-24 20:35:09.3|Trace|HttpClient|Req: [GET] http://qbittorrent:8080/api/v2/app/preferences
2026-01-24 20:35:09.3|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:09.3|Trace|HttpClient|Res: HTTP/1.1 [GET] http://qbittorrent:8080/api/v2/app/preferences: 200.OK (6043 bytes) (21 ms)
2026-01-24 20:35:09.3|Trace|HttpClient|Response content (6043 bytes): {}
2026-01-24 20:35:09.4|Trace|HttpClient|Req: [GET] http://qbittorrent:8080/api/v2/torrents/info?category=sonarr
2026-01-24 20:35:09.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2026-01-24 20:35:09.4|Trace|HttpClient|Res: HTTP/1.1 [GET] http://qbittorrent:8080/api/v2/torrents/info?category=sonarr: 200.OK (59911 bytes) (27 ms)
2026-01-24 20:35:09.4|Trace|HttpClient|Response content (59911 bytes): []
2026-01-24 20:35:09.5|Debug|Parser|Parsing string 'River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb'
2026-01-24 20:35:09.5|Trace|Parser|^(?<title>.+?)[-_. ]+?(?:S|Season|Saison|Series|Stagione)[-_. ]?(?<season>\d{1,2}(?![-_. ]?\d+))(?:[-_. ]|$)+(?<extras>EXTRAS|SUBPACK)?(?!\\)
2026-01-24 20:35:09.5|Debug|Parser|Episode Parsed. River Sunset - Season 01 
2026-01-24 20:35:09.5|Debug|Parser|Languages parsed: Unknown
2026-01-24 20:35:09.5|Debug|QualityParser|Trying to parse quality for 'River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb'
2026-01-24 20:35:09.5|Debug|Parser|Quality parsed: WEBDL-1080p v1
2026-01-24 20:35:09.5|Debug|Parser|Release Group parsed: MWeb
2026-01-24 20:35:09.5|Debug|AggregateLanguages|Language couldn't be parsed from release, fallback to series original language: Chinese
2026-01-24 20:35:09.5|Debug|AggregateLanguages|Selected languages: Chinese
2026-01-24 20:35:09.5|Debug|TrackedDownloadService|Tracking 'qBittorrent:River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb': ClientState=Downloading (readonly) SonarrStage=Downloading Episode='River Sunset - Season 01 WEBDL-1080p v1' OutputPath=.
2026-01-24 20:35:09.5|Debug|Parser|Parsing string 'River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb'
2026-01-24 20:35:09.5|Trace|Parser|^(?<title>.+?)[-_. ]+?(?:S|Season|Saison|Series|Stagione)[-_. ]?(?<season>\d{1,2}(?![-_. ]?\d+))(?:[-_. ]|$)+(?<extras>EXTRAS|SUBPACK)?(?!\\)
2026-01-24 20:35:09.5|Debug|Parser|Episode Parsed. River Sunset - Season 01 
2026-01-24 20:35:09.5|Debug|Parser|Languages parsed: Unknown
2026-01-24 20:35:09.5|Debug|QualityParser|Trying to parse quality for 'River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb'
2026-01-24 20:35:09.5|Debug|Parser|Quality parsed: WEBDL-1080p v1
2026-01-24 20:35:09.5|Debug|Parser|Release Group parsed: MWeb
2026-01-24 20:35:09.5|Debug|AggregateLanguages|Language couldn't be parsed from release, fallback to series original language: Chinese
2026-01-24 20:35:09.5|Debug|AggregateLanguages|Selected languages: Chinese
2026-01-24 20:35:09.5|Debug|TrackedDownloadService|Tracking 'qBittorrent:River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb': ClientState=Completed (readonly) SonarrStage=Ignored Episode='River Sunset - Season 01 WEBDL-1080p v1' OutputPath=/data/torrents/sonarr/River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb/River.Sunset.S01E01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb.mkv.
2026-01-24 20:35:09.5|Trace|EventAggregator|Publishing TrackedDownloadRefreshedEvent

### Anything else?

```json
{
  "added_on": 1769258104,
  "amount_left": 402563042,
  "auto_tmm": true,
  "availability": 4.001999855041504,
  "category": "sonarr",
  "comment": "1106214",
  "completed": 2801664,
  "completion_on": -1,
  "content_path": "/data/torrents/sonarr/River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb/River.Sunset.S01E02.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb.mkv",
  "dl_limit": 0,
  "dlspeed": 334080,
  "download_path": "",
  "downloaded": 1691740,
  "downloaded_session": 2833044,
  "eta": 3414,
  "f_l_piece_prio": false,
  "force_start": false,
  "has_metadata": true,
  "hash": "",
  "inactive_seeding_time_limit": -2,
  "infohash_v1": "",
  "infohash_v2": "",
  "last_activity": 1769258106,
  "magnet_uri": "magnet:?xt=urn:btih:&dn=River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb&tr=https%3A%2F%2Ftracker.m-team.cc%2F",
  "max_inactive_seeding_time": -1,
  "max_ratio": 3,
  "max_seeding_time": -1,
  "name": "River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb",
  "num_complete": 59,
  "num_incomplete": 0,
  "num_leechs": 3,
  "num_seeds": 4,
  "popularity": 0,
  "priority": 1,
  "private": true,
  "progress": 0.00691146505487826,
  "ratio": 0,
  "ratio_limit": 3,
  "reannounce": 14,
  "root_path": "/data/torrents/sonarr/River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb",
  "save_path": "/data/torrents/sonarr",
  "seeding_time": 0,
  "seeding_time_limit": -2,
  "seen_complete": 1769258107,
  "seq_dl": false,
  "size": 405364706,
  "state": "downloading",
  "super_seeding": false,
  "tags": "",
  "time_active": 3,
  "total_size": 405364706,
  "tracker": "https://tracker.m-team.cc",
  "trackers_count": 1,
  "up_limit": 0,
  "uploaded": 0,
  "uploaded_session": 0,
  "upspeed": 0
}
```
![](https://r2-test.shuiguocan.com/sonarr/1.png)
![When i click manual import](https://r2-test.shuiguocan.com/sonarr/2.png)
![](https://r2-test.shuiguocan.com/sonarr/3.png)

**Comment by stevietv** [comment]
the release it downloaded was called `River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb` - that is a name of a season pack.
The fact that it contains only one episode is down to the releaser/indexer.

Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord](https://discord.gg/Ex7FmFK), or [IRC](https://web.libera.chat/?channels=#sonarr) for support/questions.

**Comment by mynameisbogdan** [comment]
The indexer doesn't even have season packs, the issue is that the episodes are under a folder `River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb`.

![Image](https://github.com/user-attachments/assets/0fd586a2-207e-45b1-b0eb-0671c889b72b)

I guess they do this to help with season packing and cross-seeding!?! 🤷

**Comment by lhbdhr** [comment]
> The indexer doesn't even have season packs, the issue is that the episodes are under a folder `River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb`.
> 
> ![Image](https://github.com/user-attachments/assets/0fd586a2-207e-45b1-b0eb-0671c889b72b)
> 
> I guess they do this to help with season packing and cross-seeding!?! 🤷

@mynameisbogdan @stevietv 
But this is truly a serious issue that heavily impacts the Sonarr user experience. 
Is there any way to resolve this directly within Sonarr?
Hopefully, we can determine if it's a season pack based on the number of video files within the torrent.

**Comment by lhbdhr** [comment]
> The indexer doesn't even have season packs, the issue is that the episodes are under a folder `River.Sunset.S01.1080p.TX.WEB-DL.AAC2.0.H.264-MWeb`.
> 
> ![Image](https://github.com/user-attachments/assets/0fd586a2-207e-45b1-b0eb-0671c889b72b)
> 
> I guess they do this to help with season packing and cross-seeding!?! 🤷

By the way, how does Sonarr handle parsing? Does it look at the torrent's filename, the internal metainfo name, or the final folder name?

**Comment by stevietv** [comment]
sonarr parses the release name on grab,. and the folder and filenames on import.
From that folder name sonarr expects a season pack, and is warning you on the queue with the orange icons that it didn't find all the episode files that should have been in that folder.

**Comment by lhbdhr** [comment]
@stevietv 

Thank you for providing this information.

"I'd like to emphasize that this isn't just an isolated case, but rather a widespread issue for users in the Chinese-speaking community.

In many releases from Chinese groups, the **`Name` field within the torrent metadata** (the display name that determines the root folder in the download client) only contains the series title and season info (e.g., `Series.Name.S01.1080p.WEB-DL...`), even if the torrent itself contains only a single episode file.

Most Chinese release groups follow a specific naming convention where the folder name only contains the series title and season info (e.g., Series.Name.S01.1080p.WEB-DL...).

Please reconsider the reasonableness of the request I made. 
I believe the issue I raised affects many people.
